pages/workspace/tracking-plan/implementation-status.mdx

```diff
@@ -100,5 +100,7 @@ Based on these checks we report the implementation status using one of the follo
 - Event is sometimes sent correctly
 - Event is never sent correctly
 
+> Inspector monitors events from all branches, so it may report 'Sometimes sent correctly' for the correctly implemented events if old version is still sent from main or other branches.
```

Comment:
```suggestion
> Inspector monitors events from all branches, so it may report 'Sometimes sent correctly' for events that have been correctly implemented  on your branch if older versions are still being sent from other branches.
```